src/cli-utils/init-nextjs-conference.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const os = require('os');
+
+function initNextJSConference({ answers, projectPath }) {
+	// Update website .env file
+	const envVarsWebsite = {
+		NEXT_PUBLIC_CRYSTALLIZE_TENANT_IDENTIFIER: answers.tenant,
+		NEXT_PUBLIC_SERVICE_API_URL: 'https://conference-service-api.netlify.app',
+	};
+	fs.writeFileSync(
+		path.resolve(projectPath, 'website/.env'),
+		Object.keys(envVarsWebsite)
+			.map((key) => `${key}=${envVarsWebsite[key]}`)
+			.join(os.EOL) + os.EOL
+	);
+
+	// Update Service API .env.local file
+	const envVarsServiceAPI = {
+		CRYSTALLIZE_TENANT_IDENTIFIER: answers.tenant,
+		JWT_SECRET: 'some-secret-jwt-key-goes-here',
+		EMAIL_FROM: '',
+		CRYSTALLIZE_ACCESS_TOKEN_ID: '',
+		CRYSTALLIZE_ACCESS_TOKEN_SECRET: '',
+		STRIPE_SECRET_KEY: '',
+		STRIPE_PUBLISHABLE_KEY: '',
+		SENDGRID_API_KEY: '',
+		SERVICE_CALLBACK_HOST: '',
+		TILLIT_API_KEY: '',
+	};
+	fs.writeFileSync(
+		path.resolve(projectPath, 'service-api/.env.local'),
```

Comment:
Using .env.local files is a good idea. That should be used for the service API too. I think you need to update the .gitignore files to ignore `.env.*` also